app/models/translation_string.rb

```diff
@@ -75,7 +76,13 @@ def update_localization
       raise("Localization for :#{tag.to_sym} doesn't exist!")
     end
 
-    data[tag.to_sym] = text
+    store_localization
+  end
```

Comment:
This method may be orphaned now. 

Originally I had "after_update" calling this method, which calls "store_localization" after the guard clause. The problem with that is that when the db is being set up during boot or testing, it throws this error and aborts, because the localizations do not at that point exist.

Maybe a `rescue` would solve the problem? I don't understand that method, but suggestions welcome if you feel the guard is necessary.